src/Symfony/Component/Notifier/Bridge/Telnyx/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/telnyx-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Telnyx Notifier Bridge",
+    "keywords": ["sms", "telnyx", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.4|^5.0",
+        "symfony/notifier": "^5.2"
```

Comment:
Wouldn't it work with notifier 5.1 too?

Not sure how we handle this in general? 

cc @jschaedl @fabpot

Reply:
If this PR is merged, it will be part of the Symfony 5.3 release. Symfony 5.1 will be EOL by then. I wouldn't change this line.

Reply:
In any case, the code is not compatible with 5.1.

Reply:
should use `~5.3.0`

Reply:
Done